editor.planx.uk/src/@planx/components/Checklist/model.ts

```diff
@@ -1,7 +1,7 @@
 import { array } from "yup";
 
 import { BaseNodeData, Option } from "../shared";
-import { ChecklistLayout } from "./Public/components/VisibleChecklist";
+import { ChecklistLayout } from "./Public/Public";
```

Comment:
See previous comment about importing this from types.ts - avoids this slightly unexpected pattern.